dotcom-rendering/playwright/lib/load-page.ts

```diff
@@ -17,18 +17,20 @@ const loadPage = async (
 	path: string,
 	waitUntil: 'load' | 'domcontentloaded' = 'domcontentloaded',
 	region = 'GB',
+	preventSupportBanner = true,
 ): Promise<void> => {
 	await page.addInitScript((regionProvided) => {
 		// force geo region
 		window.localStorage.setItem(
 			'gu.geo.override',
 			JSON.stringify({ value: regionProvided }),
 		);
-		// prevent support banner
-		window.localStorage.setItem(
-			'gu.prefs.engagementBannerLastClosedAt',
-			`{"value":"${new Date().toISOString()}"}`,
-		);
+		if (preventSupportBanner) {
```

Comment:
the local storage item now prevents a request being sent to the API, so we need to prevent this in the banner test